api/src/mail/index.ts

```diff
@@ -86,34 +86,36 @@ export default async function sendMail(options: EmailOptions) {
 }
 
 export async function sendInviteMail(email: string, url: string) {
+	const emailCaseInsensitive = email.toLowerCase();
 	const defaultOptions = await getDefaultTemplateOptions();
 
 	const html = await liquidEngine.renderFile('user-invitation', {
 		...defaultOptions,
-		email,
+		email: emailCaseInsensitive,
```

Comment:
It doesn't really matter here if the email is lowercase or now, as it's an email w/ template

Reply:
Thank you @rijkvanzanten, I added this one by mistake.